Makefile

```diff
@@ -2,8 +2,8 @@ DOCKER_REGISTRY    ?= presidio.azurecr.io
 DOCKER_BUILD_FLAGS :=
 LDFLAGS            :=
 
-BINS        = presidio-anonymizer presidio-ocr presidio-anonymizer-image presidio-api presidio-scheduler presidio-datasink presidio-collector presidio-recognizers-store
```

Comment:
if you add to the bins line, you get a new image.
make sure it is not pushed to acr with latest/latest-dev ... it should have only numeric tag.

Reply:
was this fixed?

Reply:
As per our phone discussion, we will do it in 2 phases.
Current phase remains intact. Next phase will discuss some level of separation.